packages/ckeditor5-dev-tests/bin/test-travis.sh

```diff
@@ -19,4 +19,4 @@ yarn run lint && \
 ${ROOT_BIN}/ckeditor5-dev-tests-check-dependencies && \
 cd ${CKEDITOR5_TEST_ENVIRONMENT} && \
 node --max_old_space_size=4096 $ROOT_BIN/ckeditor5-dev-tests --files=$PACKAGE_NAME --coverage --reporter=dots --browsers=Chrome && \
-node --max_old_space_size=4096 $ROOT_BIN/ckeditor5-dev-tests --files=$PACKAGE_NAME --reporter=dots --browsers=Firefox,BrowserStack_Edge,BrowserStack_Safari
+node --max_old_space_size=4096 $ROOT_BIN/ckeditor5-dev-tests --files=$PACKAGE_NAME --reporter=dots --browsers=Firefox
```

Comment:
Or `Firefox,Chrome`?

Reply:
And in the https://github.com/ckeditor/ckeditor5-react/pull/95/files we can have Chrome too I guess.

Reply:
No, we shouldn't. We want to generate CC report only once. If we merge those lines, we will generate two reports.

Reply:
Ok, so that's fine for me.